dev/playbooks/deploy_my_machine.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+# usage:
+# http://unix.stackexchange.com/questions/10922/temporarily-suspend-bash-history-on-a-given-shell
+
+# http://stackoverflow.com/a/2654048/2975300
+
+# http://stackoverflow.com/a/28393320/2975300
+
```

Comment:
I would recommend defining `set -euo pipefail` which is the unofficial bash strict mode http://redsymbol.net/articles/unofficial-bash-strict-mode/

This will help debugging as well as help stopping the script if something fails.